### PR TITLE
Go/Web-Hv can now accept a correctly generated BugReport that include…

### PR DESCRIPTION
**Pull request by sandonian**
…s time lapse data, and show that timelapse data on the screen.

Remaining tasks include fixing the properties component of the hierarchy view screen,
adding an icon to the activity list for activities with time-lapse data, more thorough
manual testing / bug fixes, and perhaps latency optimizations.

Use this link: https://sandonian.github.io/ to test these changes.

### CODE REVIEW

File: js/ddmlib/ddmclient.js [hunk 1]
```diff
@@ -340,6 +340,26 @@ class OfflineServiceController {
...
 
+class TimeLapseBugReportServiceController {
+    constructor(appInfo) {
+        this.bugReportLine = appInfo.data
+    }
+
+    loadViewList() {
+        const base64String = this.bugReportLine.replace(VIEW_CAPTURE_REGEX, "")
+
+        return protobuf.load("../protos/view_capture.proto").then(function (root) {
```
Comment (sunnygoyal) [inline js/ddmlib/ddmclient.js:351]: This part can probably be moved to worker.js so that the UI thread is not blocked

File: js/hview.js [hunk 2]
```diff
@@ -608,57 +615,42 @@ $(function () {
...
-                    if ($("#vlist_content label").length == 0) {
-                        tVList.children().appendTo("#vlist_content")
-                        tBox.children().appendTo("#border-box")
-                        applyResizeData();
-                    }
-                }).catch(msg => { handleLoadingListError(msg) });
+        viewController.loadViewList().then(function(nodes) {
+            rootNodes = nodes
+
+            for(let i = 0; i < rootNodes.length; i++) {
```
Comment (sunnygoyal) [inline js/hview.js:639]: This loop is probably causing ANR in chrome. We can probably add a deferred logic here in a followup cl

File: js/ddmlib/ddmclient.js (hunk 1 above)
Comment (sandonian) [reply js/ddmlib/ddmclient.js:351]: I'll work on moving this code into worker.js in a follow-up CL.

File: js/hview.js (hunk 2 above)
Comment (sandonian) [reply js/hview.js:639]: It certainly is. I'll fix this in a future CL.

**Pull request merged**